spec/dotenv_spec.rb

```diff
@@ -164,7 +164,7 @@
   describe "with an instrumenter" do
     let(:instrumenter) { double("instrumenter", instrument: {}) }
     before { Dotenv.instrumenter = instrumenter }
-    after { Dotenv.instrumenter = nil }
+    after { Dotenv.instrumenter = ActiveSupport::Notifications }
```

Comment:
This is putting `instrumenter` back to the 'default' value: https://github.com/bkeepers/dotenv/blob/4c5c83ba38b9d7976f01fccd91e5ab10ab06227d/lib/dotenv/rails.rb#L17